src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+function Button({ title, disabled }) {
+
+    return (
+        <button disabled={disabled} type="button" onClick={() => console.log(title)}>{title}</button>
```

Comment:
`<button disabled={disabled}` heb je handig gedaan. Ik heb volgens mij met if statements gewerkt. Dit is natuurlijk veel netter, korter en leesbaarder. Als je het niet erg vind pik ik dit vanaf nu van je. ;P